include/mxnet/symbolic.h

```diff
@@ -327,8 +327,10 @@ class Executor {
   /*!
    * \brief Perform a Forward operation of Operator
    *  After this operation, user can get the result by using function head.
+   * \param is_train Whether this is training phase.
+   * \param need_tracer The variable indicates whether to record this forward pass.
    */
-  virtual void Forward(bool is_train) = 0;
+  virtual void Forward(bool is_train, bool need_tracer) = 0;
```

Comment:
Can we simply use TurnTracer on and off option here? without adding need_tracer?